### PR TITLE
Fix for memory leaks

### PR DESCRIPTION
**Pull request by Franc1sco**

### CODE REVIEW

File: addons/sourcemod/scripting/MyJailbreak/weapons.sp [hunk 1]
```diff
@@ -291,7 +291,13 @@ public void OnConfigsExecuted()
...
 	ListWeapons();
-
+	
+	// prevent memory leaks
+	if (g_hMenu1 != null)delete g_hMenu1;
+	if (g_hMenu2 != null)delete g_hMenu2;
+	if (g_hMenu3 != null)delete g_hMenu3;
+	if (g_hMenu4 != null)delete g_hMenu4;
+	
 	// Create menus
```
Comment (Hexer10) [inline addons/sourcemod/scripting/MyJailbreak/weapons.sp]: Afaik you don't need to check for `null` using delete you should do it only with `CloseHandle`

File: addons/sourcemod/scripting/MyJailbreak/weapons.sp [hunk 2]
```diff
@@ -291,7 +291,13 @@ public void OnConfigsExecuted()
 	g_aPrimary = CreateArray(128);
 	g_aSecondary = CreateArray(128);
```
Comment (Hexer10) [inline addons/sourcemod/scripting/MyJailbreak/weapons.sp]: I think we need to close these handles as well.

Comment (Franc1sco) [reply addons/sourcemod/scripting/MyJailbreak/weapons.sp]: here https://github.com/shanapu/MyJailbreak/pull/333

**Comment by Hexer10** [comment]
Closing this in favour of #333

**Pull request closed without merging**